model/browser/events.yaml

```
@@ -0,0 +1,83 @@
+groups:
+  - id: event.enduser.user_action
+    stability: development
+    type: event
+    name: enduser.user_action
+    brief: >
+      This event describes actions performed by the user such as click, scroll, zoom, resize, etc.
+    body:
+      id: enduser.user_action
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: element
+          type: string
+          stability: development
+          brief: Target element tag name and it is obtained via `event.target.tagName`.
+          requirement_level: recommended
+          examples: ["button"]
+        - id: xpath
+          type: string
+          stability: development
+          brief: Target element xpath
+          requirement_level: recommended
+          examples: ["//*[@id='testBtn']"]
+        - id: user_action_type
+          type: enum
+          members:
+            - id: click.left
+              value: "click.left"
+              stability: development
+              brief: An element is left clicked by a user.
+            - id: click.right
+              value: "click.right"
+              stability: development
+              brief: An element is right clicked by a user.
+            - id: click.middle
+              value: "click.middle"
+              stability: development
+              brief: An element is middle clicked by a user.
+            - id: scroll
+              value: "scroll"
+              stability: development
+              brief: An element is scrolled by a user.
+            - id: zoom
+              value: "zoom"
+              stability: development
+              brief: An element is zoomed by a user.
+            - id: resize
+              value: "resize"
+              stability: development
+              brief: An element is resized by a user.
+            - id: keyboardenter
+              value: "keyboardenter"
+              stability: development
+              brief: An element is entered via keyboard by a user.
+            - id: keyboardspace
+              value: "keyboardspace"
```

just curious, why is space special compared to other characters?

does `keyboard.enter` not include space?

```suggestion
            - id: keyboard.space
              value: "keyboard.space"
```

---

Updated, Thanks Trask!